ssw/ACS-Z9100/portmap.ini

```diff
@@ -0,0 +1,32 @@
+1=1,1
```

Comment:
This was needed during our earlier testing of sfputil, however, I believe that with the changes in Azure/sonic-buildimage#316, this can be removed.